_rules/heading-accessible-name-ffd0e9.md

```diff
@@ -0,0 +1,153 @@
+---
+id: ffd0e9
+name: Heading has accessible name
+rule_type: atomic
+description: |
+  This rule checks that each heading has an accessible name.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:2.4.6: # Headings and Labels (AA)
```

Comment:
I don't think this rule is testing this SC. You're just checking that there is an accessible name, not that it describes the topic or purpose of the section it heads.

Reply:
@EmmaJP's argument was that if a heading has empty name, it cannot describe its topic. Hence failing this rule (having an empty heading) also fails 2.4.6 (the heading is not descriptive). The fail-on-fail is enough to be listed as requirement. As always, **passing** this rule does not mean anything for the SC.

https://github.com/act-rules/act-rules.github.io/pull/400#discussion_r346930882

Reply:
That's fair enough

Reply:
Yep, this rule could be an atomic part of a composite rule for 2.4.6